src/store/annotations/util.ts

```diff
@@ -0,0 +1,31 @@
+import * as uuid from 'uuid';
+import { Annotation } from '../../@types';
+
+export function addClientIds(annotation: Annotation): Annotation {
```

Comment:
Maybe this could be moved to the `drawingUtils`? Creation of drawings will also need to generate these client ids, so perhaps this could be applied to a `PathGroup[]` instead of `Annotation`?